README.md

```diff
@@ -56,19 +56,19 @@ Live Server
 ### Setup
 
 Clone this repository to your desired folder: 
-git@github.com:Gmanlove/Hello-Microverse.git
+https://github.com/tomdan-ai/Hello-Microverse.git
 
 
 ## 👥 Authors <a name="authors"></a>
 
 👤 **Author1**
 
-- GitHub: [Gmanlove](https://github.com/Gmanlove)
+- GitHub: [tomdan-ai](https://github.com/tomdan-ai)
 
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-I would like to thank Stephen Okhaishie
+I would like to thank Emmnauel Obi
 
 
```

Comment:
- [ ] Kindly include 1-3 features of this project in the **_Key Features_** section of your README.md file. You can include learning how to setup linters, learning how to git ignore files and so on. 👍